app/src/main/java/org/simple/clinic/home/patients/PatientScreenRepository.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.home.patients
+
+import io.reactivex.Observable
+import org.simple.clinic.home.patients.illustration.HomescreenIllustration
+import org.simple.clinic.storage.files.FileStorage
+import javax.inject.Inject
+
+class PatientScreenRepository @Inject constructor(
```

Comment:
I feel that `PatientScreenRepository` may not be the right name for this. It seems to me that the only reason this is being called `PatientScreenRepository` is because it is being used in the `PatientsScreen`.

Currently, we follow the convention of naming a repository around the database model (i.e., `Appointment.RoomDao` is managed by `AppointmentRepository`). Can we follow this convention and name it `HomescreenIllustrationRepository`?

Reply:
In addition, I feel this should go into the `illustration` package.

Reply:
Done.